src/fragments/lib/auth/common/mfa_preferences/common.mdx

```diff
@@ -0,0 +1,37 @@
+<Callout>
+Fetching and Updating MFA preferences is only available through the escape hatch APIs under the AWS Cognito Auth plugin. 
```

Comment:
I thought the escape hatch was different. Should this say:

```suggestion
The APIs for fetching and updating MFA preferences are only available on the AWS Cognito Auth plugin and not on the Auth category.
```

Reply:
Updated.